<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking</class>
 <widget class="QDialog" name="Booking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Booking{border-image: url(:/images/seat.jpeg);}
*{
font: 700 12pt &quot;Nexa Bold&quot;;
}

#frame{
background-color: rgba(255, 255, 255,0.8);
border-style: outset;
border-width: 1px;
border-radius: 10px;
border-color: black;
}
QPushButton{
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}
QLabel{
color:black;
font: 700 15pt &quot;Nexa Bold&quot;;
}
QPushButton:hover{
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 0, 0);
background-color: rgb(0,0,0);
color: white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>550</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>110</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Islamabad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lahore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Karachi</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>110</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Islamabad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lahore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Karachi</string>
     </property>
    </item>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumDate">
     <date>
      <year>1752</year>
      <month>10</month>
      <day>14</day>
     </date>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>50</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Passengers</string>
    </property>
   </widget>
   <widget class="QLabel" name="serror">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Please select source</string>
    </property>
   </widget>
   <widget class="QLabel" name="derror">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Please select destination</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateerror">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Please select date</string>
    </property>
   </widget>
   <widget class="QLabel" name="paserror">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Please select Number of Passengers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
